server/api/orders.js

```diff
@@ -0,0 +1,28 @@
+const router = require('express').Router();
+const Order = require('../db/models/Order');
+
+router.post('/', async (req, res, next) => {
+  try {
+    const newOrder = await Order.create(req.body);
```

Comment:
You might want to validate some information about who the session belongs to, and what is in this `req.body`.